bootstrap-extensions/src/main/java/de/agilecoders/wicket/extensions/markup/html/bootstrap/form/tempusdominus/TempusDominusConfig.java

```diff
@@ -260,8 +261,8 @@ public TempusDominusConfig withViewDate(Date date) {
     /**
      * @return Date format pattern set for TempusDominusLocalizationConfig.getFormat()
      */
-    public String getFormat() {
-        return localization.getFormat();
+    public String getJavaFormat() {
```

Comment:
Can we keep the old name ? For backward compatibility
E.g. `getFomat()` could be `@Deprecated` and delegating to `getJavaFormat()`

Reply:
done!
Thanks for the review :)